utils/loadFontAsync.ios.js

```diff
@@ -0,0 +1,28 @@
+import { NativeModules } from 'react-native';
+let { EXFontLoader } = NativeModules;
+let _FONTS$ = {};
+
+/**
+ * Load a font asynchronously from a url.
+ *
+ * NOTE: On iOS, we can infer the font family name and style
+ * from the downloaded font, so a URL is all we need to
+ * do the load.
+ */
+export default function loadFontAsync({ uri }) {
```

Comment:
let's make sure we post a warning here if we are missing props that are required for it to work on Android @skevy (as per Slack discussion)